Update warehouses.md - added section about the users table (#5207)
segmentio/segment-docs: src/engage/warehouses.md

| ------- | ---------------------- | ---------------------- |
| u123    | 450.00                 | total_revenue_180_days |

## Users table

> There are a couple of nuances to keep in mind when it comes to the users table. This table is an aggregate view based on the `user_id` field. This means that anonymous profiles with just an `anonymous_id` identifier aren't included in this view. You can still view identify calls for anonymous audiences and computed traits in the `identifies` table. 

Reviewer: ```suggestion
The users table is an aggregate view based on the `user_id` field. This means that anonymous profiles with just an `anonymous_id` identifier aren't included in this view. You can still view identify calls for anonymous audiences and computed traits in the `identifies` table. 
```

> Users table will be synced as soon as the warehouse is connected as a destination in Engage, if you have previously created Engage computations. As a result, the table might contain the data from computations not directly connected to the warehouse.

Reviewer: ```suggestion
The users table is synced as soon as the warehouse is connected as a destination in Engage, if you've previously created Engage computations. As a result, the table might contain data from computations not directly connected to the warehouse.
```
